rstblog/config.py

```diff
@@ -39,7 +39,7 @@ def list_entries(self, key):
         for layer in self.stack:
             for key, value in layer.iteritems():
                 if key.startswith(prefix):
-                    rv[key] = value
+                    rv[key[len(prefix):]] = value
```

Comment:
I don't know that this won't have unintended consequences somewhere else, but it looks OK from my vantage.